lib/rubocop/options.rb

```diff
@@ -247,6 +248,7 @@ module OptionsHelp
       debug:                'Display debug info.',
       display_cop_names:    'Display cop names in offense messages.',
       display_style_guide:  'Display style guide URLs in offense messages.',
+      extra_details:        'Display extra details in offense messages.',
```

Comment:
not sure I follow the example `Display Details configuration parameter in offense messages.` but yes add to README ...